soc/arm/st_stm32/stm32l0/Kconfig.defconfig.stm32l051xx

```diff
@@ -0,0 +1,14 @@
+# ST Microelectronics STM32L051XX MCU
+
+# Copyright (c) 2021 Even Falch-Larsen <even.falch.larsen@nomono.co>
```

Comment:
Copyright is different from other ones . Normal ?
`Copyright (c) 2021 Nomono AS`

Reply:
Fixed.